blocks/library/latest-posts/index.php

```diff
@@ -13,13 +13,18 @@
  * @return string Returns the post content with latest posts added.
  */
 function gutenberg_render_block_core_latest_posts( $attributes ) {
-	$recent_posts = wp_get_recent_posts( array(
-		'numberposts' => $attributes['postsToShow'],
-		'post_status' => 'publish',
-		'order'       => $attributes['order'],
-		'orderby'     => $attributes['orderBy'],
-		'category'    => $attributes['categories'],
-	) );
+	$recent_posts = wp_get_recent_posts(
+		apply_filters(
+			'widget_posts_args',
+			array(
+				'numberposts' => $attributes['postsToShow'],
+				'post_status' => 'publish',
+				'order'       => $attributes['order'],
+				'orderby'     => $attributes['orderBy'],
+				'category'    => $attributes['categories'],
+			)
```

Comment:
The `widget_posts_args` filter in core takes two params `$args` and `$instance`. You only have one arg here. 

See https://developer.wordpress.org/reference/hooks/widget_posts_args/

At least it should copy the `$args` array to prevent plugins from triggering a warning due to an a `null` variable.

Reply:
This is presently known. I'll look into a fix :+1: